Cargo.toml

```diff
@@ -74,3 +74,7 @@ harness = false
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[patch."crates-io"]
+opentelemetry = { git = "https://github.com/djc/opentelemetry-rust", branch = "tracer-sampler" }
+opentelemetry_sdk = { git = "https://github.com/djc/opentelemetry-rust", branch = "tracer-sampler" }
```

Comment:
These probably shouldn't be here anymore?

Reply:
Yup, got rid of them.